crates/bifrost/src/providers/replicated_loglet/tasks/seal.rs

```diff
@@ -140,6 +140,8 @@ impl<T: TransportConnect> SealSingleNode<T> {
             match self.do_seal().await {
                 Ok(res) if res.body().sealed || res.body().status == Status::Ok => {
                     let _ = tx.send((self.node_id, res.body().local_tail));
+                    self.known_global_tail
+                        .notify_offset_update(res.body().header.known_global_tail);
```

Comment:
Just to make sure I got this correctly. This is to make sure Clients that initiated the seal still learn about the last global tail know by the log server?

Reply:
That's correct.